sympy/plotting/tests/test_plot.py

```diff
@@ -699,3 +703,454 @@ def test_issue_20113():
+    assert r[0][3] == "test"
+
+    # multiple expressions
+    args = _plot_sympify((x + y, x * y))
+    r = _check_arguments(args, 1, 2)
+    assert isinstance(r, (list, tuple, Tuple)) and len(r) == 2
+    assert len(r[0]) == 4
+    assert len(r[1]) == 4
+    assert r[0][0] == x + y
+    assert r[0][1] == (x, -10, 10) or (y, -10, 10)
+    assert r[0][2] == (y, -10, 10) or (x, -10, 10)
+    assert r[0][1] != r[0][2]
+    assert r[0][3] == "x + y"
+    assert r[1][0] == x * y
+    assert r[1][1] == (x, -10, 10) or (y, -10, 10)
+    assert r[1][2] == (y, -10, 10) or (x, -10, 10)
+    assert r[1][1] != r[0][2]
+    assert r[1][3] == "x*y"
+
+    # multiple expressions, same custom ranges
+    args = _plot_sympify((x + y, x * y, (x, -2, 2), (y, -4, 4)))
+    r = _check_arguments(args, 1, 2)
+    assert isinstance(r, (list, tuple, Tuple)) and len(r) == 2
+    assert len(r[0]) == 4
+    assert len(r[1]) == 4
+    assert r[0][0] == x + y
+    assert r[0][1] == (x, -2, 2)
+    assert r[0][2] == (y, -4, 4)
+    assert r[0][1] != r[0][2]
+    assert r[0][3] == "x + y"
+    assert r[1][0] == x * y
+    assert r[1][1] == (x, -2, 2)
+    assert r[1][2] == (y, -4, 4)
+    assert r[1][1] != r[0][2]
+    assert r[1][3] == "x*y"
+
+    # multiple expressions, custom ranges and labels
+    args = _plot_sympify([(x + y, (x, -2, 2), (y, -4, 4)),
+                (x * y, (x, -3, 3), (y, -6, 6), "test")])
+    r = _check_arguments(args, 1, 2)
+    assert isinstance(r, (list, tuple, Tuple)) and len(r) == 2
+    assert len(r[0]) == 4
+    assert len(r[1]) == 4
+    assert r[0][0] == x + y
+    assert r[0][1] == (x, -2, 2)
+    assert r[0][2] == (y, -4, 4)
+    assert r[0][1] != r[0][2]
+    assert r[0][3] == "x + y"
+    assert r[1][0] == x * y
+    assert r[1][1] == (x, -3, 3)
+    assert r[1][2] == (y, -6, 6)
+    assert r[1][1] != r[0][2]
+    assert r[1][3] == "test"
+
+
+    ### Test arguments for plot3d_parametric_surface()
+
+    # single parametric expression
+    args = _plot_sympify((x + y, cos(x + y), sin(x + y)))
+    r = _check_arguments(args, 3, 2)
+    assert isinstance(r, (list, tuple, Tuple)) and len(r) == 1
+    assert len(r[0]) == 6
+    assert r[0][0] == x + y
+    assert r[0][1] == cos(x + y)
+    assert r[0][2] == sin(x + y)
+    assert r[0][3] == (x, -10, 10) or (y, -10, 10)
+    assert r[0][4] == (y, -10, 10) or (x, -10, 10)
+    assert r[0][3] != r[0][4]
+    assert r[0][5] == "(x + y, cos(x + y), sin(x + y))"
+
+    # single parametric expression, custom ranges and labels
+    args = _plot_sympify((x + y, cos(x + y), sin(x + y), (x, -2, 2),
+                (y, -4, 4), "test"))
+    r = _check_arguments(args, 3, 2)
+    assert isinstance(r, (list, tuple, Tuple)) and len(r) == 1
+    assert len(r[0]) == 6
+    assert r[0][0] == x + y
+    assert r[0][1] == cos(x + y)
+    assert r[0][2] == sin(x + y)
+    assert r[0][3] == (x, -2, 2)
+    assert r[0][4] == (y, -4, 4)
+    assert r[0][3] != r[0][4]
+    assert r[0][5] == "test"
```

Comment:
Why not just this:
```python
assert r == [[x + y, cos(x+y), sin(x+y), (x, -2, 2), (y, -4, 4), "test"]]
```

Reply:
Eheh I added them one at a time as I was testing, then one copy-paste led to the other... I'll compact them.

Reply:
Done